pkg/config/config.go

```diff
@@ -676,6 +686,14 @@ func ExtractConfigData(yamlData []byte) (common.Configuration, error) {
 		config.TLS.ACMEFolder = strings.Trim(config.TLS.ACMEFolder, "./")
 	}
 
+	if len(config.TLS.CertificatePath) >= 0 && strings.HasPrefix(config.TLS.CertificatePath, "./") {
+		config.TLS.CertificatePath = strings.Trim(config.TLS.CertificatePath, "./")
```

Comment:
are we assuming here that they will give a local path that starts with `./` and not a absolution path that would start with `/` , then that assumption is not correct.